infrastructure/cdn-in-a-box/optional/docker-compose.debugging.yml

```diff
@@ -62,13 +62,13 @@ services:
       args:
         TRAFFIC_MONITOR_RPM: traffic_monitor/traffic_monitor.rpm
     command: /usr/bin/true
-  # The trafficops-go-nondebug service ensures that the trafficops-go base image
-  # exists before trying to build trafficops-go-debug.
-  trafficops-go-nondebug:
-    image: trafficops-go
+  # The trafficops-nondebug service ensures that the trafficops base image
+  # exists before trying to build trafficops-debug.
+  trafficops-nondebug:
+    image: trafficops
     build:
-      context: .
-      dockerfile: traffic_ops/Dockerfile-go
+      context: ../../
+      dockerfile: traffic_ops/Dockerfile
```

Comment:
Building `trafficops-nondebug` fails:
```dockerfile
Building trafficops-nondebug
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/user/go/src/github.com/apache/trafficcontrol/traffic_ops/Dockerfile: no such file or directory
Service 'trafficops-nondebug' failed to build
```

You'll need to change the rpm path, too.

Reply:
The RPM path in `infrastructure/cdn-in-a-box/traffic_ops/Dockerfile` is `infrastructure/cdn-in-a-box/traffic_ops/traffic_ops.rpm` - I think that's correct?

Reply:
Yep

Reply:
Both `dockerfile` and `TRAFFIC_OPS_RPM` can have the same values as the `trafficops` service in `docker-compose.yml` has. It only needs to be specified again because the service name changes.